client/src/components/common/sliderCard/index.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Button from '../Button';
+import Star from '../Star';
+import './style.css';
+
+const SliderCard = ({
+  imageSrc,
```

Comment:
We agreed to make the image a static for this sprint 
so you can import the image and use it without taking it from the props